### PR TITLE
Add comdex asset

### PR DESCRIPTION
**Pull request by akhilkumarpilli**

### CODE REVIEW

**Review by IbrarMakaveli — APPROVED** [review]

**Comment by sunnya97** [comment]
Thank you guys!  Can you also add an entry to the cosmos chain registry?  https://github.com/cosmos/chain-registry/

**Comment by dheerajkd30** [comment]
> Thank you guys! Can you also add an entry to the cosmos chain registry? https://github.com/cosmos/chain-registry/

Added in the registry. FYI PR # - https://github.com/cosmos/chain-registry/pull/65
Let me know if more details are required to merge both PRs.

Thanks.

**Review by IbrarMakaveli — CHANGES_REQUESTED** [review]
Can you also add cmdx.png image please : https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/cmdx.png
Thanks !

**Comment by akhilkumarpilli** [comment]
Image was already included in file changes. [This url](https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/cmdx.png) will work once this PR got merged. Thank you.

**Review by IbrarMakaveli — APPROVED** [review]

**Comment by IbrarMakaveli** [comment]
Oh Yep, I got confused with another project, thank you very much!

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -656,6 +656,35 @@
...
+      "symbol": "CMDX",
+      "ibc": {
+        "source_channel": "channel-1",
+        "dst_channel": "channel-87",
+        "source_denom": "ucmdx"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/cmdx.png"
+      },
+      "coingecko_id": "cmdx"
```
Comment (sunnya97) [inline osmosis-1/osmosis-1.assetlist.json:687]: I don't see comdex listed on coingecko yet

Comment (anilCSE) [reply osmosis-1/osmosis-1.assetlist.json:687]: Yeah, can we keep it empty for now?

File: osmosis-1/osmosis-1.assetlist.json [hunk 2]
```diff
@@ -656,6 +656,35 @@
...
+      "display": "cmdx",
+      "symbol": "CMDX",
+      "ibc": {
+        "source_channel": "channel-1",
+        "dst_channel": "channel-87",
+        "source_denom": "ucmdx"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/cmdx.png"
+      },
```
Comment (sunnya97) [inline osmosis-1/osmosis-1.assetlist.json:686]:
```suggestion
      }
```

File: osmosis-1/osmosis-1.assetlist.json (hunk 1 above)
Comment (sunnya97) [reply osmosis-1/osmosis-1.assetlist.json:687]:
```suggestion
```

**Comment by dheerajkd30** [comment]
@akhilkumarpilli could you please update the logo to https://github.com/comdex-official/logos/blob/main/Comdex-logo-main.png

**Pull request merged**